src/goto-instrument/cover_basic_blocks.cpp

```diff
@@ -15,6 +15,26 @@ Author: Peter Schrammel
 #include <util/message.h>
 #include <util/string2int.h>
 
+/// create list of covered lines as CSV string and set as property of source
+/// location of basic block, compress to ranges if applicable
+static void update_covered_lines(cover_basic_blockst::block_infot &block_info);
```

Comment:
I think you could just define this here rather than forward-declare it?

Reply:
Yes it was mostly to avoid moving too much code, have a smaller diff and make it easier to review

Reply:
Which is a Good Thing, anyway. It's much easier to just glance at the definitions at the top of the file than to scroll through the whole thing trying to create a mental map of its contents. Forward declarations are there exactly so we don't have to reposition the function every time we want to use it before it's declared. I'd leave it as it is.